src/interface/web/app/components/navMenu/navMenu.tsx

```diff
@@ -128,7 +128,7 @@ export default function NavMenu(props: NavMenuProps) {
                                 <DropdownMenuSeparator />
                                 <DropdownMenuLabel>Profile</DropdownMenuLabel>
                                 <DropdownMenuItem>
-                                    <Link href="/config">Settings</Link>
+                                    <Link href="/configure">Configure</Link>
```

Comment:
I prefer settings for the settings page. Follows more standard UX nomenclature.

Reply:
Makes sense. Updated